scaler/index.js

```diff
@@ -163,10 +165,8 @@ exports.scaleSpannerInstancePubSub = async (pubSubEvent, context) => {
 // For testing with: https://cloud.google.com/functions/docs/functions-framework
 exports.scaleSpannerInstanceHTTP = async (req, res) => {
   try {
-    const payload =
-    '{"units": "PROCESSING_UNITS", "minSize":100, "maxSize":500, "stepSize":100,"overloadStepSize":500,"scaleOutCoolingMinutes":5,"scaleInCoolingMinutes":30,"scalingMethod":"STEPWISE","projectId":"spanner-scaler","instanceId":"autoscale-test","scalerPubSubTopic":"projects/spanner-scaler/topics/test-scaling","metrics":[{"name":"high_priority_cpu","threshold":65,"value":85, "margin": 15},{"name":"rolling_24_hr","threshold":90,"value":70},{"name":"storage","threshold":75,"value":80}],"currentNodes":0,"currentSize":100,"regional":true}';
+    const spanner = createSpannerParameters({});
```

Comment:
While this is a much nicer way of injecting test data doesn't this require `sinon` to be included in the production set of dependencies?

Reply:
yes, you're right.
I extracted the sample params to a JSON file and eliminated the dependency.